docs/docs/articles/test-workflows.md

```diff
@@ -279,32 +279,46 @@ or on the Step level:
 Files directly in `working dir`
 ```yaml
 - name: Saving artifacts
-    workingDir: /data/artifacts
-    artifacts:
-      paths:
-      - '*'
+  workingDir: /data/artifacts
+  artifacts:
+    paths:
+    - '*'
 ```
 
 Files in directories inside `working dir``
 ```yaml
 - name: Saving artifacts
-    workingDir: /data/artifacts
-    artifacts:
-      paths:
-      - '**/*'
+  workingDir: /data/artifacts
+  artifacts:
+    paths:
+    - '**/*'
 ```
 
 Artifacts can also be configured for project directories (inside `/data/repo`):
 
 ```yaml
 steps:
 - name: Saving artifacts
-    workingDir: /data/repo/test/cypress/executor-tests/cypress-13/cypress/videos
-    artifacts:
-      paths:
-      - '**/*'
+  workingDir: /data/repo/test/cypress/executor-tests/cypress-13/cypress/videos
+  artifacts:
+    paths:
+    - '**/*'
+```
+
+### condition: always
+It's a common thing to save artifacts in case of failure. That's when setting `condition: always` may be needed - to execute artifacts saving step even if the previous ones failed.
```

Comment:
By default the `artifacts` will have `condition: always` for the same level. i.e.

```yaml
- shell: abc
  artifacts:
    paths: ["**/*"]
```

is equivalent of:
```yaml
- steps:
  - shell: abc
  - condition: always
    artifacts:
      paths: ["**/*"]
```

Reply:
I added different versions with additional descriptions